tsconfig.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Leaving blank for now, details are in the tutorial linked below and we will continually add/subtract from this config as well as we decide which TS features we want to include.